### PR TITLE
Fix link for Ticket as search result

### LINKED ISSUE #239: Broken link search results
On search results, it appears that if the comment is inside a ticket, the link does not work. 
https://retroachievements.org/searchresults.php?s=CoolCanuck&o=50 (try and view the last result on this page, it links to https://retroachievements.org/7 )

### PR DESCRIPTION
**Pull request by juherr**
Fix #239

### CODE REVIEW

**Comment by meleu** [comment]
On my tests it returns nothing. :(

**Comment by juherr** [comment]
@meleu What do you mean by "nothing"?

-> SQL error ?
-> No more search results at all?
-> Wrong link on all results?

File: lib/database/search.php [hunk 1]
```diff
@@ -44,13 +44,14 @@ function performSearch($searchQuery, $offset, $count, &$searchResultsOut)
...
-                )
-            )
+        CASE
+            WHEN c.articletype=1 THEN CONCAT( '/Game/', c.ArticleID )
+            WHEN c.articletype=2 THEN CONCAT( '/Achievement/', c.ArticleID )
+            WHEN c.articletype=3 THEN CONCAT( '/User/', ua.User ),
+            WHEN c.articletype=5 THEN  CONCAT( '/feed.php?a=', c.ArticleID )
+            ELSE c.articletype
         )
+        END
```
Comment (luchaos) [inline lib/database/search.php:54]: much better, thank you :)

File: lib/database/search.php [hunk 2]
```diff
@@ -44,13 +44,14 @@ function performSearch($searchQuery, $offset, $count, &$searchResultsOut)
...
-        IF( c.articletype=1, CONCAT( '/Game/', c.ArticleID ),
-            IF( c.articletype=2, CONCAT( '/Achievement/', c.ArticleID ),
-                IF( c.articletype=3, CONCAT( '/User/', ua.User ),
-                    IF( c.articletype=5, CONCAT( '/feed.php?a=', c.ArticleID ), c.articletype )
-                )
-            )
+        CASE
+            WHEN c.articletype=1 THEN CONCAT( '/Game/', c.ArticleID )
+            WHEN c.articletype=2 THEN CONCAT( '/Achievement/', c.ArticleID )
+            WHEN c.articletype=3 THEN CONCAT( '/User/', ua.User ),
```
Comment (luchaos) [inline lib/database/search.php:50]: The trailing comma here seems to be the culprit for search not returning any results anymore

**Comment by luchaos** [comment]
Works as intended without the trailing comma in the query. Thanks!

File: lib/database/search.php (hunk 2 above)
Comment (juherr) [reply lib/database/search.php:50]: Good catch, thank!!

**Pull request merged**